.github/workflows/gluten.yml

```diff
@@ -64,6 +68,18 @@ jobs:
             git clone https://github.com/oap-project/gluten.git
             cd gluten
             ./dev/buildbundle-veloxbe.sh
+            THIRDPARTY_LIB="./package/target/thirdparty-lib"
+            LINUX_OS=$(. /etc/os-release && echo ${ID})
+            VERSION=$(. /etc/os-release && echo ${VERSION_ID})
+            ARCH=`uname -m`
+            mkdir -p $THIRDPARTY_LIB
+            cp /usr/lib/${ARCH}-linux-gnu/{libre2.so.9,libdouble-conversion.so.3,libidn.so.12,\
```

Comment:
why should we do such copy? `/usr/lib/${ARCH}-linux-gnu` should be included in `ld.so.conf`

Reply:
Build and test are run in different vm. Libraries only exists in build vm.